web/pom.xml

```diff
@@ -157,8 +157,20 @@
         <filtering>true</filtering>
         <excludes>
           <exclude>*.EXAMPLE</exclude>
+          <!-- don't alter self signed ssl keys (used for dev) -->
```

Comment:
This is useful for testing with self signed certificates so we can test against the live version